modules/howtos/pages/observability-metrics.adoc

```diff
@@ -19,6 +19,10 @@ Metrics broadly fall into the following categories:
 
 Right now only the first category is implemented by the SDK, more are planned.
 
+[NOTE]
+====
+The `AggregatingMeter`, which was previously available in SDK 3.1, has been renamed to `LoggingMeter` in the 3.2 release. The class is marked with `@Stability.Volatile` and may be subject to further change in later releases.
```

Comment:
Just double checking this with @daschl, I'm unsure if this should still be considered "Developer Preview" since the class is marked as Volatile.

Reply:
Looks like `LoggingMeter` shouldn't be marked as volatile, so I will remove any mention of this. 
Bug raised on the SDK side: https://issues.couchbase.com/browse/JVMCBC-1007